isPlandromRecursive.py

```diff
@@ -0,0 +1,24 @@
+# Define a recursive procedure is_palindrome, that takes as input a string, and returns a
+# Boolean indicating if the input string is a palindrome.
+
+# Base Case: '' => True
+# Recursive Case: if first and last characters don't match => False
+# if they do match, is the middle a palindrome?
+
+def is_palindrome(s):
+    if s == '':
+        return  True
+    if s[0] != s[len(s)-1]:
+        return False
+    if s[0] == s[len(s)-1]:
+        new_str = ''
+        for i in range(1, len(s)-1):
+            new_str += s[i]
+        return is_palindrome(new_str)
```

Comment:
You can directly get the substrings using syntax `s[from_index: to_index]` whereas from_index is inclusive and to_index in exclusive